### DISCUSSION TITLE
Create Listing with no Creator fees using Sdk

### DISCUSSION THREAD

**Discussion by wainesiro** [ProjectOpenSea/opensea-js#1366] [Q&A] [answered]
Hello, I would like to know how do i create a Listing using opensea Sdk (createListing) and not pay any creator fees?

**Comment by ryanio** [comment, marked as answer]
I can add a parameter to the function `excludeOptionalCreatorFees`. If you excluded any required creator fees, then the order would not be shown on OpenSea

**Reply by wainesiro** [reply]
that would be amazing! when can it be available?. Also didnt understand the part "the order would not be shown on Opensea"

**Reply by ryanio** [reply]
some collections have required creator fees, you can check with the `getCollection` response field in "fees" if `required` is true or false. you have to provide the required fees, otherwise the order will not be shown on opensea.

I'll try to merge and release [this PR](https://github.com/ProjectOpenSea/opensea-js/pull/1367) on Monday! it adds a param to exclude non-required creator fees for the collection

**Reply by ryanio** [reply]
this is released now https://github.com/ProjectOpenSea/opensea-js/releases/tag/v7.0.8

**Reply by wainesiro** [reply]
Thank you Fam
